modules/sdk-coin-osmo/test/unit/getBuilderFactory.ts

```diff
@@ -0,0 +1,6 @@
+import { TransactionBuilderFactory } from '../../src';
+import { coins } from '@bitgo/statics';
+
+export const getBuilderFactory = (coin: string): TransactionBuilderFactory => {
+  return new TransactionBuilderFactory(coins.get(coin));
+};
```

Comment:
Is this a utility class which can exist elsewhere ?

Reply:
removed it.